.travis.yml

```diff
@@ -1,18 +1,33 @@
-language: php
-
-php:
-  - 7.2
-  - nightly
-
 matrix:
+  include:
+    - language: php
+      php: 7.2
+      os: linux
+      before_script:
+        - composer install
+      after_script:
+        - sh .travis.coverage.sh
+      env:
+        - COVERAGE_FLAGS="--coverage-text --coverage-clover=clover.xml"
+    - language: php
+      php: nightly
+      before_script:
+        - composer install
+    - name: "PHP: 7.2"
+      language: sh
+      os: windows
+      before_script:
```

Comment:
Curious if Travis supports color output )
```
-  SET ANSICON=121x90 (121x90)
```

Reply:
Lets see xD

Reply:
Looks not like it ^^